curriculum/challenges/english/06-quality-assurance/advanced-node-and-express/use-a-template-engines-powers.md

```diff
@@ -12,15 +12,38 @@ One of the greatest features of using a template engine is being able to pass va
 
 In your Pug file, you're able to use a variable by referencing the variable name as `#{variable_name}` inline with other text on an element or by using an equal sign on the element without a space such as `p=variable_name` which assigns the variable's value to the p element's text.
 
-We strongly recommend looking at the syntax and structure of Pug [here](https://github.com/pugjs/pug) on GitHub's README. Pug is all about using whitespace and tabs to show nested elements and cutting down on the amount of code needed to make a beautiful site.
+ Pug is all about using whitespace and tabs to show nested elements and cutting down on the amount of code needed to make a beautiful site. Read the Pug documentation for more information on usage and syntax.
+ 
+ Here is an example: 
```

Comment:
That is my bad, I guess I remove the new line when I made the suggestion. There should be a blank line after "Here is an example:" and before the code block.

https://github.com/updownpress/markdown-lint/blob/master/rules/031-blanks-around-fences.md

Reply:
dub